### DISCUSSION TITLE
Let's keep the ball rolling 💣

### DISCUSSION THREAD

**Discussion by francocipollone** [Ekumen-OS/andino#9] [General]
### Intro

Creating this thread in order to debate how the carpincho effort could be continued! 
**For those who are interested in doing so! It is not mandatory of course!**

Thinking on options on how to continue this: **The one that came up to my mind and liked the most:**

### Proposal

**-->** _It would be nice to meet up once a month, at HQ (for those who are close), and allocate a block of 3 or 4hs per meeting._ 

<img src=https://user-images.githubusercontent.com/53065142/210079796-892aba4a-679e-42dc-b934-e5a372512dd2.png width=300>

### Let's pull this off

I know it sounds a lot, but it is just once a month.
By doing so we guarantee to be able to finish and polish all carpincho-related code in probably 3-4 months. Maybe less if we "manijeamos" with all of this (which will probably happen to me)

Wdy say team?  @jballoffet @BarceloChristian @LolaSegura @gdiaz-guevara @FlorGrosso

**Comment by francocipollone** [comment]
CC: @Martin-MT

**Reply by Martin-MT** [reply]
I'm up for it, but I'm not sure if it's possible to do it (it's a lot of time!)

**Reply by francocipollone** [reply]
It's ok! Thanks for answering. This was written when the water were calm, before this being part of Robocraft and Conferences. So I think that we all are cheaping from our places, so the ball is still rolling. :D
Also we have a google space for this, so things are going on just right.
